tools/CouchDB_Import_Instruments.php

```diff
@@ -85,9 +85,42 @@ function UpdateDataDicts($Instruments)
         }
     }
 
-    function generateDocumentSQL($instrument)
+    function generateDocumentSQL($instrument, $JSONData)
     {
-        return "SELECT c.PSCID, s.Visit_label, f.Administration, f.Data_entry, f.Validity, CASE WHEN EXISTS (SELECT 'x' FROM conflicts_unresolved cu WHERE i.CommentID=cu.CommentId1 OR i.CommentID=cu.CommentId2) THEN 'Y' ELSE 'N' END AS Conflicts_Exist, CASE ddef.Data_entry='Complete' WHEN 1 THEN 'Y' WHEN NULL THEN 'Y' ELSE 'N' END AS DDE_Complete, i.* FROM $instrument i JOIN flag f USING (CommentID) JOIN session s ON (s.ID=f.SessionID) JOIN candidate c ON (c.CandID=s.CandID) LEFT JOIN flag ddef ON (ddef.CommentID=CONCAT('DDE_', f.CommentID)) WHERE f.CommentID NOT LIKE 'DDE%' AND s.Active='Y' AND c.Active='Y'";
+        $select = "SELECT 
+                        c.PSCID, 
```

Comment:
this is just reformatting the query and removing all references to "instrument" tables